### ISSUE TITLE
[Cadence] Support all service publishing types with the frontend service

### ISSUE THREAD

**Issue by pregnor** [banzaicloud/banzai-charts#1284]
**Is your feature request related to a problem? Please describe.**

Currently the Cadence frontend service only supports the [service publishing types](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) `clusterIP` and `nodePort`.

**Describe the solution you'd like to see**

Add support for `loadBalancer`, `externalName`, `ingress` service publishing types to support more use cases.

**Describe alternatives you've considered**

-

**Additional context**

**Comment by longquanzheng** [comment]
I don't quite follow, isn't it supported here? https://github.com/banzaicloud/banzai-charts/blob/d8d883e026bf1b492b1aeb89383eab8105e5ab2b/cadence/templates/server-service.yaml#L14 
Because I've used this helmchart to deploy frontend with LoadBalancer and it works fine.

**Comment by pregnor** [comment]
You are right, it does support `loadBalancer` configuration with ephemeral IPs.
`loadBalancerIP` configuration option may be considered in the future.

---

FTR `externalName` configuration is required to support `externalName` service publishing and the support ingress configuration has to be investigated.
